docs/codeql/codeql-cli/about-codeql-packs.rst

```diff
@@ -82,14 +82,14 @@ The following properties are supported in ``qlpack.yml`` files.
           codeql/cpp-all: ^0.0.2
 
      - Packs that define CodeQL package dependencies on other packs
-     - A map from pack references to the semantic version range that is compatible with this pack. Supported from CLI version v2.6.0 and onwards.
+     - A map from pack references to the semantic version range that is compatible with this pack. Supported for CLI versions v2.6.0 and later.
    * - ``defaultSuiteFile``
      - .. code-block:: yaml
 
         defaultSuiteFile: cpp-code-scanning.qls
 
      - Packs that export a set of default queries to run
-     - The path to a query suite file containing all of the queries that are run by default when this pack is passed to the ``codeql database analyze`` command. Supported from CLI version v2.6.0 and onwards.
+     - The path to a query suite file relative to the package root, containing all of the queries that are run by default when this pack is passed to the ``codeql database analyze`` command. Supported from CLI version v2.6.0 and onwards. Only one of ``defaultSuiteFile`` or ``defaultSuite`` can be defined.
```

Comment:
Nit: we say something like "Supported from CLI version v2.6.0 and onwards." in a few different ways throughout.  We could make it more consistent, e.g. using "Supported by CLI versions v2.6.0 and later." throughout.

Reply:
I'm a little torn about including versioning at all.

1. If we are including a version constraint for this feature, we should be including it for all packaging-related features, and all other new features.
2. How far back do we need to maintain documentation for? 
3. In 2.6.0, the implementation was not complete, so even though this feature was supported, I would still recommend using a later version if you really wanted to use packaging.
4. We don't have any published packs supported by 2.6.0. I think the first packs in the registry were available from 2.7.3.

@jf205 can you give some advice on this? 

Perhaps, we can do a final pass after the docs are written and add version constraints to features where appropriate.

Reply:
You're right, adding versions for this is tricky. It might be less confusing for users if we simply update the reusable beta note to include a more up to date version of the CLI. If users really want to check what's supported in a really old version of the CLI they can check the changelog.